src/lib/drupal/staticPaths.ts

```diff
@@ -55,6 +55,14 @@ export async function getStaticPathsByResourceType(
   )
 
   // Convert the resources to static paths
-  const paths = drupalClient.buildStaticPathsFromResources(modifiedResources)
+  //  TODO: This should probably not use a Drupal-specific utility to work on post-formatted data.
+  //  This is currently using a type assertion to make it possible since we know that the necessary
+  //  props will be present, though they probably shouldn't be present because the formatted data
+  //  prop names are a bit specific to drupal (e.g. `pid` on Path).
```

Comment:
i'm not 100% following this this early, `modifiedResources as Parameters<
      typeof drupalClient.buildStaticPathsFromResources
    >[0]` does look a bit funky

Reply:
This ideally wouldn't be here, as noted in the code comment. More or less, we're just telling the Typescript compiler that the thing is shaped like the object expected by the function to which the thing is being passed.

Reply:
I'm changing this to implement the conversion without next-drupal. This will go away.